prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -690,6 +690,18 @@ void rxr_pkt_handle_send_error(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entr
 			 * completion. In all other cases, we write an eq entry because
 			 * there is no application operation associated with handshake.
 			 */
+			char ep_addr_str[OFI_ADDRSTRLEN], peer_addr_str[OFI_ADDRSTRLEN];
+			size_t buflen=0;
```

Comment:
Add an empty line here between declaration and implementation.

Reply:
Done.